src/resources/ApiKeys/ApiKeys.ts

```diff
@@ -27,8 +27,11 @@ export default class ApiKey extends Resource {
         return this.api.put(path, apiKey);
     }
 
-    delete(apiKeyId: string) {
-        return this.api.delete(`${ApiKey.baseUrl}/${apiKeyId}`);
+    delete(apiKeyIds: string | string[]) {
+        if (Array.isArray(apiKeyIds) && apiKeyIds.length > 1) {
+            return this.api.post<void>(`${ApiKey.baseUrl}/delete/bulk`, apiKeyIds);
```

Comment:
It should be a delete, not a post and the path is not the correct one: https://platform.cloud.coveo.com/docs?urls.primaryName=AuthorizationServer#/Api%20Keys/rest_organizations_paramId_apikeys_bulk_delete
```suggestion
            return this.api.delete<void>(`${ApiKey.baseUrl}/bulk`, apiKeyIds);
```

Reply:
I asked for it to be changed to a POST, since adding content to the body of a DELETE call is not a practice 100% supported by the HTTP spec.

Here's what [the spec](https://www.rfc-editor.org/rfc/rfc9110.html#name-delete) says:
> Although request message framing is independent of the method used, content received in a DELETE request has no generally defined semantics, cannot alter the meaning or target of the request, and might lead some implementations to reject the request and close the connection because of its potential as a request smuggling attack ([Section 11.2](https://www.rfc-editor.org/rfc/rfc9112#section-11.2) of [[HTTP/1.1](https://www.rfc-editor.org/rfc/rfc9110.html#HTTP11)]). A client SHOULD NOT generate content in a DELETE request unless it is made directly to an origin server that has previously indicated, in or out of band, that such a request has a purpose and will be adequately supported.